target/linux/ar71xx/files/arch/mips/ath79/Kconfig.openwrt

```diff
@@ -1943,3 +1943,14 @@ config ATH79_MACH_TL_WPA8630
 	select ATH79_DEV_LEDS_GPIO
 	select ATH79_DEV_M25P80
 	select ATH79_DEV_WMAC
+
+config ATH79_MACH_AP531B0
+	bool "Rockeetech AP531B0 board support"
+	select SOC_QCA953X
+	select ATH79_DEV_GPIO_BUTTONS
```

Comment:
Please, reorder all `select ATH79...` alphabetically, as in many other boards in this file.
You have also changed file attributes (`100644 → 100755`) which is **wrong** - please, fix it.